shopelectro/tests/tests_selenium.py

```diff
@@ -375,6 +383,33 @@ def test_load_more_after_filtering(self):
 
         self.assertEqual(new_product_cards, 96)
 
+    def test_cached_cart(self):
+        """
+        The cart state shouldn't be cached on a category page.
+
+        @todo #783:60m Lazily load the cart.
+         It will solve the issue with invalid cart data and
+         bring ability to create cache shared among users.
+        """
+        def add_products_from(url):
+            self.browser.get(url)
+            self.wait_page_loading()
+            for i in range(3):
+                self.add_to_cart(i)
+
+        add_products_from(self.root_category)
+        add_products_from(self.children_category)
+        final_total = cart_total(self.browser)
+
+        self.browser.get(self.root_category)
+        self.wait_page_loading()
+        self.browser.get(self.children_category)
+        self.wait_page_loading()
```

Comment:
now test is not failed. It means, that it doesn't check the problem with the task.

I'm not sure, but maybe to reproduce bug behavior from the task, you should remove this two lines:
```
        self.browser.get(self.children_category)
        self.wait_page_loading()
```

Reply:
Hm, it is failing on my PC. Probably CI settings effect on the result

Reply:
@artemiy312 , i've not launched it. I meant that the test should be decorated with `expectedFailure` in case it's failing